src/overlays/actors/ovl_En_Dnh/z_en_dnh.h

```diff
@@ -10,7 +10,20 @@ typedef void (*EnDnhActionFunc)(struct EnDnh*, PlayState*);
 typedef struct EnDnh {
     /* 0x000 */ Actor actor;
     /* 0x144 */ EnDnhActionFunc actionFunc;
-    /* 0x148 */ char unk_148[0x74];
+    /* 0x148 */ SkelAnime skelAnime;
+    /* 0x18C */ u16 unk18C;
+    /* 0x18E */ char pad18E[2];
+    /* 0x190 */ s32 unk190;
+    /* 0x194 */ s32 unk194;
+    /* 0x198 */ s16 unk198;
+    /* 0x19A */ char pad19A[2];
+    /* 0x19C */ s16 unk19C;
+    /* 0x19E */ s16 unk19E;
+    /* 0x1A0 */ s32 unk1A0;
+    /* 0x1A4 */ Vec3s unk1A4;
+    /* 0x1AA */ char pad1AA[6];
+    /* 0x1B0 */ Vec3s unk1B0;
+    /* 0x1B6 */ char pad1B6[6];
```

Comment:
We know from `SkelAnime_Init(` that `unk1A4` is a jointTable and that `unk1B0` is the morph table, which are array's of `Vec3s`, not single values. We also know from the last arg of `SkelAnime_Init(` that these are array's of size 2`, so that's where the padding comes from. A `Vec3s` is of size 6, so 
```c
    /* 0x1A4 */ Vec3s unk1A4;
    /* 0x1AA */ char pad1AA[6]
```
is really
```c
    /* 0x1A4 */ Vec3s jointTable[2];
```